README.md

```diff
@@ -1 +1,61 @@
-# warehouse-server
\ No newline at end of file
+Project structure:
+
+├── warehouse-server
+│   ├── controllers - Defines app routes logic.
+│   │   ├── currentUser.controllers.js
+│   │   └── product.controllers.js
+│   │   ├── sender.controllers.js
+│   │   └── shop.controllers.js
+│   │   ├── user.controllers.js.js
+│   │   └── sessoin.controllers.js
+│   ├── middlewares -Checking logged in status middlware.
+│   │   ├── isUserMiddlewares.js
+│   ├── database - Connect database and server.
+│   │   ├── connection.js
+│   ├── routes - All routes for different entities in different files.
+│   │   ├── currentUser.routers.js
+│   │   ├── product.routers.js
+│   │   └── sender.routers.js
+│   │   ├── sessions.routers.js
+│   │   ├── shop.routers.js
+│   │   └── user.routers.js
+│   ├── db
+│   │   ├── migrations - Contains the application migration data.
+│   │   │   ├── 20191024185314-create-user.js
+│   │   │   ├── 20191024190446-create-product.js
+│   │   │   ├── 20191024190536-create-shop.js
+│   │   │   └── 20191024190705-create-sender.js
+│   │   ├── models - Contains database models.
+│   │   │   ├── index.js
+│   │   │   ├── product.js
+│   │   │   └── sender.js
+│   │   │   ├── shop.js
+│   │   │   └── user.js
+|   ├── errorsCollection - For diffeent cases errors collection.
+│   │   └── error.js
+|   ├── config - Contains application configuration settings such as database configuration.
+│   │   └── config.json
+|   ├── package.json
+|   ├── README.md
+|   ├── app.js
+|   ├── sequelizeMeta.json
```

Comment:
Structure should not ends like this, see Armine Read.me example.
Also where is `package-lock.json` ?